paypal/express/views.py

```diff
@@ -445,6 +486,8 @@ def render_to_response(self, methods, basket):
         else:
             # No shipping methods available - we flag this up to PayPal indicating that we
             # do not ship to the shipping address.
-            pairs.append(('NO_SHIPPING_OPTION_DETAILS', 1))
+            pass
+            # pairs.append(('NO_SHIPPING_OPTION_DETAILS', 1))
         payload = urlencode(pairs)
+        logger.info(payload)
```

Comment:
Looks like another logger for debug.

Reply:
changed to logger.debug() as I believe this is useful debugging info.